src/app/components/SubmitButton/SubmitButton.tsx

```diff
@@ -1,13 +1,29 @@
 import React from 'react';
 import type { ReactNode } from 'react';
 import styles from './SubmitButton.module.css';
+import Typography from '../Typography/Typography';
 
 type SubmitButtonProps = {
+  icon:
+    | 'ArrowRight'
+    | 'Camera'
+    | 'Letter'
+    | 'Safe'
+    | 'Search'
+    | 'Send'
+    | 'Signup'
+    | 'Upload';
   children: ReactNode;
 };
 
-export default function submitButton({
+export default function SubmitButton({
+  icon,
   children,
 }: SubmitButtonProps): JSX.Element {
-  return <button className={styles.submitButton}>{children}</button>;
+  return (
+    <button className={styles.submitButton}>
+      <img src={`src/assets/SVGs/${icon}.svg`} className={styles.icon} />
```

Comment:
Good solution if you want to avoid using React components as icons :)
Maybe something for another week 👍